surfman/Cargo.toml

```diff
@@ -89,3 +89,4 @@ winapi = { version = "0.3", features = ["d3d11", "wingdi", "winuser", "libloader
 
 [target.'cfg(target_os = "android")'.dependencies]
 "raw-window-handle" = "0.3.3"
```

Comment:
We should change this to match the version winit uses, so that rust doesn't complain about the HasRawWindowHandle trait not being implemented for Window type.

Reply:
How did you generate the warning?

Reply:
It was a rustc error message that I got when I tried building locally without updating the version of raw-window-handle here.

You can build locally using:
```
rustup target add "arm-linux-androideabi"
cd surfman
cargo build --target "arm-linux-androideabi"
```